network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -534,10 +534,11 @@ class ProductPage(AirtableMixin, FoundationMetadataPageMixin, Page):
     )
 
     # How does it use this data?
-    how_does_it_use_data_collected = models.TextField(
+    how_does_it_use_data_collected = RichTextField(
         max_length=5000,
+        features=['bold', 'italic', 'link'],
```

Comment:
let's file a follow-up to unify this set somewhere, too, because if more than two fields all need it, that's "a property of a set of fields" rather than "hardcoded values".